app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeActivity.kt

```diff
@@ -224,13 +224,13 @@ class ComposeActivity :
         val mediaAdapter = MediaPreviewAdapter(
             this,
             onAddCaption = { item ->
-                CaptionDialog.newInstance(item.localId, item.description, item.uri)
-                    .show(supportFragmentManager, "caption_dialog")
+                CaptionDialog.newInstance(item.localId, item.description, item.uri).show(supportFragmentManager, "caption_dialog")
             },
             onAddFocus = { item ->
                 makeFocusDialog(item.focus, item.uri) { newFocus ->
                     viewModel.updateFocus(item.localId, newFocus)
                 }
+                // TODO this is inconsistent to CaptionDialog (device rotation)?
```

Comment:
?

Reply:
There was a change/issue to make the text in the alt text dialog survive a device rotation.

This was not done to the focus dialog.

So one could change this as well (which is this todo).